lib/lita/handlers/jenkins.rb

```diff
@@ -35,11 +35,11 @@ def self.default_config(config)
       def jenkins_build(response)
         requested_job = response.matches.last.last
 
-        if requested_job.number?
-          job = jobs[requested_job.to_i - 1]
-        else
-          job = jobs.select { |j| j['name'] == requested_job }.last
-        end
+        job = if requested_job.number?
+                jobs[requested_job.to_i - 1]
+              else
+                jobs.select { |j| j['name'] == requested_job }.last
+              end
```

Comment:
@miketheman For what it's worth I really like changes like this that clean up the code. On a related note, I much prefer this syntax over the ternary version. It's much more readable to me than say

`requested_job.number? ? jobs[requested_job.to_i - 1] : jobs.select { |j| j['name'] == requested_job }.last`

anyway, :+1: